spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -131,6 +131,29 @@
     "defaultValue": true
   },
   {
+    "name": "spring.data.neo4j.indexes.auto",
```

Comment:
There is no need for that. IDE should extract the javadoc from the enum. Your IDE may not do that yet but we have no plan to do that ourselves.